### PR TITLE
Add wiki page about toggling progression system

### PR DESCRIPTION
**Pull request by ASpoonPlaysGames**
Not rly sure that this is needed, but it'll probably be useful for a /tag thing or something?

Images used in the page, thanks github for hosting these
Untitled

Untitled

### CODE REVIEW

**Comment by uniboi** [comment]
maybe not *everything* needs to be documented

**Comment by ASpoonPlaysGames** [comment]
> maybe not _everything_ needs to be documented

Look, I was asked to make this page

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
   * [Commands](using-northstar/commands.md)
   * [Launch arguments](using-northstar/launch-arguments.md)
   * [Advanced](using-northstar/advanced.md)
+  * [Progression System](using-northstar/progression.md)
```
Comment (GeckoEidechse) [inline docs/SUMMARY.md]: I'm gonna switch the order of the two. Advanced feels like the sorta thing that should show up last :P

```suggestion
  * [Progression System](using-northstar/progression.md)
  * [Advanced](using-northstar/advanced.md)
```

File: docs/using-northstar/progression.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Progression
+
+As of version `1.19.0` Northstar allows you to toggle the progression system.
+This means that you can choose whether to have everything unlocked or not.
```
Comment (GeckoEidechse) [inline docs/using-northstar/progression.md:4]: Gonna add mention that by default everything is unlocked

```suggestion
This means that you can choose whether to have everything unlocked or not.
By default progression is disabled and all weapons and skins are unlocked.
```

**Review by GeckoEidechse — COMMENTED** [review]
Two small changes. Gonna apply them myself real quick ^^

File: docs/SUMMARY.md (hunk 1 above)
Comment (ASpoonPlaysGames) [reply docs/SUMMARY.md]: Fair enough

**Review by GeckoEidechse — COMMENTED** [review]
Applied changes from review. Merging as I'm planning to release this soon :D

**Pull request merged**